Dmitry_Kononkov/06.Ansible.Workshop/roles/nginx/files/index.html

```diff
@@ -0,0 +1,8 @@
+<html>
+    <head>
+        <title>Welcome to Test.com!</title>
```

Comment:
Please convert to template, no need to duplicate one file with almost same content

title can be generated by variable